src/scratchtocatrobat/converter/converter.py

```diff
@@ -592,6 +600,46 @@ def converted_project_for(cls, scratch_project, progress_bar=None, context=None)
         assert catrobat.is_background_sprite(catrobat_project.getDefaultScene().getSpriteList().get(0))
         return ConvertedProject(catrobat_project, scratch_project)
 
+    def _add_bricks_for_visible_variables(self, catrobat_scene):
+        scratch_project = self.scratch_project
+        brick_dict = dict()
+        sprite_dict = dict()
+        sprite_list = catrobat_scene.getSpriteList()
+        for sprite in sprite_list:
+            sprite_name = sprite.getName()
+            # TODO: rewrite when there is a general name for the background sprite in the code
+            if sprite_name == "Hintergrund":
+                sprite_name = "Stage"
+            sprite_dict[sprite_name] = sprite
+        for var in scratch_project._var_to_visibility_map:
+            if(scratch_project._var_to_visibility_map.get(var)[0]):
+                target_sprite = scratch_project._var_to_visibility_map.get(var)[1]
+                sprite_object = sprite_dict[target_sprite]
+                var_object = _ScratchObjectConverter._catrobat_scene.getDataContainer().getUserVariable(var, sprite_object)
+                show_variable_brick = catbricks.ShowTextBrick(self.visible_var_X, self.visible_var_Y)
+                show_variable_brick.setUserVariableName(var)
+                show_variable_brick.setUserVariable(var_object)
+                if target_sprite not in brick_dict.keys():
+                    brick_dict[target_sprite] = []
+                brick_dict[target_sprite].append(show_variable_brick)
+            self.visible_var_Y -= self.visible_var_position_step_Y
+            if self.visible_var_Y <= self.visible_var_position_threshold_Y:
+                self.visible_var_Y = self.visible_var_Y_init
+                self.visible_var_X += self.visible_var_position_step_X
+                if self.visible_var_X >= self.visible_var_position_threshold_X:
+                    log.info("Too many visible variables")
+                    break
+        for sprite in sprite_list:
+            sprite_name = sprite.getName()
+            # TODO: rewrite when there is a general name for the background sprite in the code
+            if sprite_name == "Hintergrund":
+                sprite_name = "Stage"
+            if sprite_name in brick_dict.keys():
+                when_started_script = catbase.StartScript()
+                when_started_script.getBrickList().addAll(brick_dict[sprite_name])
+                sprite.addScript(when_started_script)
+        return
```

Comment:
You can remove this line. Empty `return` statements in Python are fairly uncommon.